applications/asset_tracker/README.rst

```diff
@@ -121,33 +153,32 @@ After programming the application and all prerequisites to your board, test the
 1. Connect the board to the computer using a USB cable.
```

Comment:
Yes, that is valid for both boards. The intention is that all text is valid for all boards, unless otherwise noted.

Connecting the Thingy:91 to a terminal emulator entails the same operations as with the DK.